.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
+    timezone: Europe/Berlin
+  open-pull-requests-limit: 10
```

Comment:
No reason to have limit in our case I would say.

Reply:
Thanks. Fixed the limit. 😄